### PR TITLE
Feature/usepip

### PR DESCRIPTION
**Pull request by specialunderwear**
Nolonger needs debianize.sh
This now has the tools to base the buildout plugin on

### CODE REVIEW

File: vdt/versionplugin/debianize/shared.py [hunk 1]
```diff
@@ -88,9 +90,13 @@ def __init__(self, version, args, extra_args, directory):
         self.directory = directory
         self.exit_code = 0
 
+    def update_exit_code(self, code):
+        if self.exit_code == 0:
+            self.exit_code = code
+
```
Comment (maerteijn) [inline vdt/versionplugin/debianize/shared.py]: Is this because you want to catch the first exit code you encounter (below) which is other than 0?

Comment (specialunderwear) [inline vdt/versionplugin/debianize/shared.py]: I want it not to go back to 0 snap you, if is error not go 0 nomore

Comment (maerteijn) [inline vdt/versionplugin/debianize/shared.py]: Then it should be:
``` python
if code !=0:
    self.exit_code = code
```
right?

Comment (specialunderwear) [inline vdt/versionplugin/debianize/shared.py]: echt not, only when the code is not an error code I must change it to error.

**Pull request merged**